applications/ShapeOptimizationApplication/python_scripts/model_part_controller_factory.py

```diff
@@ -65,6 +66,14 @@ def __init__(self, model_settings, model):
         self.damping_regions = {}
         self.damping_utility = None
 
+        self.shell_thickness_adaptor = None
+        if self.model_settings["shell_thickness_adaption"].GetBool():
+            import shell_thickness_adaptor
```

Comment:
```suggestion
            from . import shell_thickness_adaptor
```
should work, can you try please?

otherwise:
`from KratosMultiphysics.ShapeOptimizationApplication.shell_thickness_adaptor import ShellThicknessAdaptor`

Reply:
first version 
```
ImportError: attempted relative import with no known parent package
```
second version works. Why is it preferrable over 
```
from shell_thickness_adaptor import ShellThicknessAdaptor
```
?

Reply:
~~~py
from shell_thickness_adaptor import ShellThicknessAdaptor
~~~
will no longer work with the final version of the python-interface of Kratos, bcs then the folder `python_scripts` will no longer be added to the path

we should change all "direct" imports over time, then we won't have problems when we change the python interface

If you want more info then please talk to me in person :)

Reply:
:+1: